internal/constants.go

```diff
@@ -26,10 +26,10 @@ const (
 	DatabaseName         = "application-service"
 	CorrelationHeaderKey = "X-Correlation-ID"
 
-	ApiTriggerRoute   = clients.ApiBase + "/trigger"
-	ApiV2TriggerRoute = v2.ApiBase + "/trigger"
-	ApiSecretsRoute   = clients.ApiBase + "/secrets"
-	ApiV2SecretsRoute = v2.ApiBase + "/secrets"
+	ApiTriggerRoute     = clients.ApiBase + "/trigger"
+	ApiV2TriggerRoute   = v2.ApiBase + "/trigger"
+	ApiSecretsRoute     = clients.ApiBase + "/secrets"
```

Comment:
not v2?

Reply:
That one is for V1, so it doesn't change. Will be ripped out by the time we release Ireland

Reply:
got it